students/Kenneth_Murray/lesson3/slicing/slicing.py

```diff
@@ -0,0 +1,92 @@
+def exchange_first_last(seq):
+	"""returns a sequence with the first and last characters swapped"""
+	first = seq[0]
+	mid = seq[1:-1]
+	last = seq[-1]
+	
+	if type(seq) == str:
+		return last + mid + first
+	else:
+		eos=len(seq)
```

Comment:
what does `eos` stand for?  Best to spell out the all the words rather than acronyms.  Other developers won't know what your are thinking.  Best to use descriptive variable names.